tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2763,6 +2767,7 @@ <h2>Show similarity to selected datapoint</h2>
 
       selectedIndicesChanged_: function(event) {
         this.selected = event.detail.value;
+        this.setAttributions();
```

Comment:
no action required: Instead of having imperative calls, it might be better to have a observer to `this.selected` for this instead.